etna/pipeline/mixins.py

```diff
@@ -20,18 +18,16 @@
 class ModelPipelinePredictMixin:
     """Mixin for pipelines with model inside with implementation of ``_predict`` method."""
 
-    ts: Optional[TSDataset]
     model: ModelType
     transforms: Sequence[Transform]
 
-    def _create_ts(self, start_timestamp: pd.Timestamp, end_timestamp: pd.Timestamp) -> TSDataset:
+    def _create_ts(self, ts: TSDataset, start_timestamp: pd.Timestamp, end_timestamp: pd.Timestamp) -> TSDataset:
         """Create ``TSDataset`` to make predictions on."""
-        self.ts = cast(TSDataset, self.ts)
-        df = self.ts.raw_df.copy()
+        df = ts.raw_df.copy()
         # we make it through deepcopy to handle df_exog=None
-        df_exog = deepcopy(self.ts.df_exog)
-        freq = self.ts.freq
-        known_future = self.ts.known_future
+        df_exog = deepcopy(ts.df_exog)
+        freq = ts.freq
+        known_future = ts.known_future
```

Comment:
1. We haven't used `predict` in notebooks yet.
2. Ok, I'll add smth.
3. We don't require anything about transformations from given `ts` because inside `predict` it is reconstructed to its original state (look at `_create_ts`).